esp-hal/CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - ESP32-S3: `PsramConfig::core_clock` is now an `Option` (#3974)
 - `RtcSlowClock::RtcFastClock8m` has been renamed to `RtcFastClock::RtcFastClockRcFast` (#3993)
 - `RtcSlowClock::RtcSlowClockRtc` has been renamed to `RtcSlowClock::RtcSlowClockRcSlow` (#3993)
+- DMA buffer size calculation now uses `div_ceil` instead of manual ceiling division (#4022)
```

Comment:
Wouldn't say this is strictly necessary but it doesn't hurt

Reply:
What Dominic says is that the changelog isn't necessary

Reply:
Oh great, sorry for the misunderstanding.